packages/manager/src/features/IAM/Shared/AssignedPermissionsPanel/AssignedPermissionsPanel.style.ts

```diff
@@ -0,0 +1,27 @@
+import { Box, Paper, styled, Typography } from '@linode/ui';
+
+export const StylesPaper = styled(Paper)(({ theme }) => ({
```

Comment:
Nit: I think this is a typo and meant to be `StyledPaper`

```suggestion
export const StyledPaper = styled(Paper)(({ theme }) => ({
```

Reply:
Thanks for the review!
I've updated this.